src/test/java/core/basesyntax/imp/FruitAddTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.imp;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecordDto;
+import core.basesyntax.model.Storage;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitAddTest {
+    private static final int AMOUNT = 12;
+    private static final String NAME_FRUIT = "banana";
+    private static Fruit fruit;
+    private static FruitAdd fruitAdd;
+    private static FruitRecordDto fruitRecordDto;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fruitAdd = new FruitAdd();
+        fruit = new Fruit(NAME_FRUIT);
+    }
+
+    @After
+    public void afterEachTest() {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void checkAddOperation_ok() {
+        fruitRecordDto = new FruitRecordDto("b", fruit, AMOUNT);
```

Comment:
Instead of creating it over and over again In every method, use setUp method or beforeClass method

Reply:
I need it